pkg/jobs/jobspb/jobs.proto

```diff
@@ -271,7 +271,11 @@ message RestoreDetails {
   // DebugPauseOn describes the events that the job should pause itself on for debugging purposes.
   string debug_pause_on = 20;
 
-  // NEXT ID: 22.
+  // ImportIntoDatabases stores the descriptors that are being restored to
+  // but not being restored themselves. They're needed to map tables to a
+  // public schema ID.
+  repeated sqlbase.DatabaseDescriptor import_into_databases = 22;
```

Comment:
If we know that this database already exists, can we not write the `ParentSchemaID` in the descriptor rewrite to point to the material public schema ID of the database we are restoring into? Then during job execution, we only need to remap for databases without material public schema descs that are being restored.

Reply:
Good call, removed.